index.html

```diff
@@ -1,52 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why was this file deleted?

Reply:
I renamed it to home-page so I think git just recognizes as deleted.